pom.xml

```diff
@@ -12,6 +12,7 @@
     <!-- List of modules -->
     <modules>
         <!-- Add here the paths of all modules (e.g. <module>modules/MyModule</module>) -->
+        <module>modules/Twitter</module>
```

Comment:
I don't know of any other plugin called "Twitter" so this could work, but otherwise consider being a bit more specific. For example, "TwitterStreamingImporter" (and updating the folders, etc. to use this new name)